packages/roosterjs-editor-plugins/lib/Paste/wacConverter/convertPastedContentFromWac.ts

```diff
@@ -0,0 +1,128 @@
+import { HtmlSanitizer } from 'roosterjs-html-sanitizer';
+
+const WORD_ONLINE_IDENTIFYING_SELECTOR = "div.ListContainerWrapper,div.OutlineElement";
```

Comment:
I'm afraid this check is too simple. Any other app can also use these class names. Can we add more robust check?